packages/manager/modules/support/src/new-ticket/new-ticket.controller.js

```diff
@@ -40,7 +46,17 @@ export default class SupportNewController {
   getTicketURL(id) {
     return this.$state.href('support.tickets.ticket', {
       id,
-      cleanCache: true,
+    });
+  }
+
+  getTicketsURL() {
+    return this.$state.href('support.tickets');
+  }
+
+  fetchURLS() {
+    this.OvhApiMe.v6().get().$promise.then((me) => {
```

Comment:
Could it make sense to inject the `me` through the resolve instead ? It will avoid adding loading and we can possibly inherit from some root states which avoids multiple API calls, what do you think ?